index.js

```diff
@@ -14,4 +14,128 @@ cloSerX.addEventListener('click', () => {
 navListItems.forEach((item) => item.addEventListener('click', () => {
   mariMarzi.classList.add('hide');
   mariMarzi.classList.remove('mobile-menu');
-}));
\ No newline at end of file
+}));
+function createPopupWindow(project) {
+  const popupWindow = document.createElement('div');
+  popupWindow.classList.add('popup-window');
+  popupWindow.innerHTML = `<div>
+      <a id="close-popup"><img src="files/Icon.svg" alt="Close Button"></a>
+      <h2 class="card-header">${project.name}</h2>
+      <ul class="card-badges"></ul>`;
+  project.techs.forEach((item) => {
+    popupWindow.querySelector('ul').innerHTML += `<li>${item}</li>`;
+  });
+  popupWindow.querySelector('div').innerHTML += `
+      <div class="img-placeholder"><img src="${project.imgSrc}" alt="The project preview"></div>
```

Comment:
- [ ] I suggest that you should always leave spaces between code blocks. This improves readability. A space between line 27 & 28 would server as a clear indicator to mark where one block ends & another begins. What do you think 🤔